### ISSUE TITLE
Translate `library/urllib.request.po`

### ISSUE THREAD

**Issue by jordanSu** [python/python-docs-zh-tw#146]
Translate `library/urllib.request.po`
This one is relatively big, is it ok to translate all the contents in one PR or should I separate into several PRs?

Progress:

- [x] urllib.request 的 functions
- [x] urllib.request 的 classes (part 1, until `urllib.request.Request`)
- [ ] urllib.request 的 classes (part 2, all other classes) **(WIP)**
- [ ] Request Objects
- [ ] OpenerDirector 物件
- [ ] BaseHandler 物件
- [ ] HTTPRedirectHandler 物件、HTTPCookieProcessor 物件、ProxyHandler 物件
- [ ] HTTPPasswordMgr 物件、HTTPPasswordMgrWithPriorAuth 物件
- [ ] AbstractBasicAuthHandler 物件、HTTPBasicAuthHandler 物件、ProxyBasicAuthHandler 物件
- [ ] AbstractDigestAuthHandler 物件、HTTPDigestAuthHandler 物件、ProxyDigestAuthHandler 物件
- [ ] HTTPHandler 物件、HTTPSHandler 物件、FileHandler 物件、DataHandler 物件、FTPHandler 物件
- [ ] CacheFTPHandler 物件、UnknownHandler 物件、HTTPErrorProcessor 物件
- [ ] Examples
- [ ] Legacy interface
- [ ] urllib.request Restrictions
- [ ] urllib.response --- Response classes used by urllib

**Comment by mattwang44** [comment]
@jordanSu Breaking it into multiple smaller PRs is recommended, but one larger PR is also valid.

**Comment by jordanSu** [comment]
Progress:

- [ ] urllib.request 的 functions
- [ ] urllib.request 的 classes (part 1, until `urllib.request.Request`)
- [ ] urllib.request 的 classes (part 2, all other classes)
- [ ] Request Objects
- [ ] OpenerDirector 物件
- [ ] BaseHandler 物件
- [ ] HTTPRedirectHandler 物件、HTTPCookieProcessor 物件、ProxyHandler 物件
- [ ] HTTPPasswordMgr 物件、HTTPPasswordMgrWithPriorAuth 物件
- [ ] AbstractBasicAuthHandler 物件、HTTPBasicAuthHandler 物件、ProxyBasicAuthHandler 物件
- [ ] AbstractDigestAuthHandler 物件、HTTPDigestAuthHandler 物件、ProxyDigestAuthHandler 物件
- [ ] HTTPHandler 物件、HTTPSHandler 物件、FileHandler 物件、DataHandler 物件、FTPHandler 物件
- [ ] CacheFTPHandler 物件、UnknownHandler 物件、HTTPErrorProcessor 物件
- [ ] Examples
- [ ] Legacy interface
- [ ] urllib.request Restrictions
- [ ] urllib.response --- Response classes used by urllib

**Comment by jordanSu** [comment]
#162 already working on:
- urllib.request 的 functions
- urllib.request 的 classes (part 1, until `urllib.request.Request`)

**Comment by mattwang44** [comment]
@jordanSu Would you like to keep translating `library/urllib.request.po`? Just want to make sure you're aware that letting somebody else handle the rest is totally fine.

**Comment by jordanSu** [comment]
@mattwang44 Yes I would like to continue translating this file, will work on **urllib.request 的 classes (part 2, all other classes)**
